entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -e
+
+workflows $@ worker --automigrate &
+exec workflows "$@" server --automigrate
```

Comment:
Entrypoint should force you to run `workflows`, otherwise you won't be able to run anything else (eg. bash to debug). Let's just remove it, as we don't need it at this stage.
In any case, we won't run server and worker in the same container, we'll always use two different containers (or PODs in Kubernetes).

Reply:
@alfrunes sorry, follow up; after inspecting other mender components I believe keeping the same "style" is important, this I suggest to use as entry-point.sh:

```
#!/bin/sh -e
exec workflows $@
```

Reply:
Hmm.. I think `workflows` can be treated differently as it's basically two components in a single application. I was conflicted how I should make the entrypoint as I wanted to default to forking both processes. Now that I come to think of it, I like your suggestion above - leave out the entrypoint and simply default to spawning the server (CMD).